README.md

```diff
@@ -31,6 +31,24 @@ Also:
 
 ## Anvil
 
+### Dependencies
+You'll need to install the following dependencies; note, that those package
+names may vary depending on your OS and linux distribution:
+- `libdbus-glib-1`
```

Comment:
```suggestion
```
Since we migrated to `libseat` fully, we no longer depend on dbus directly, so this can be dropped.

Reply:
aye aye captain!

Reply:
done